applications/fishing-map/src/features/map-features/map-features.hooks.ts

```diff
@@ -0,0 +1,136 @@
+import { useSelector, useDispatch } from 'react-redux'
```

Comment:
why do we need `map` and `map-features`? what are the differences?

Reply:
`map-features` is for stuff that is overlaid on top of the map, not strictly necessary but that way we don't end up with gigantic map slice and map selectors and map component.